openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -82,22 +85,37 @@ static CpuArchitecture getCpuArchitecture(final String cpuArchName) {
         }
     }
 
+    static LibcImplementation getLibcImplementation(final String libcImpl) {
+        if (libcImpl == null || libcImpl.isEmpty()) {
+            logger.debug("FDZ_OPENML_JAVA_LIBC not set, assuming glibc as libc implementation.");
```

Comment:
Instead of repeating the variable name, we should put it as a static field / constant...